components/tools/OmeroJava/test/integration/gateway/TablesFacilityTest.java

```diff
@@ -171,6 +175,7 @@ public void testInvalidParams() throws Exception {
         }
     }
 
+    @Test(dependsOnMethods = { "testInvalidParams" })
```

Comment:
This should be reverted. This test doesn't actually have any method dependencies.